README.md

```diff
@@ -23,7 +23,7 @@ As no book is perfect, you will likely come by issues and might have some questi
 * Follow [@survivejs](https://twitter.com/survivejs) at Twitter for official news or poke me through [@bebraw](https://twitter.com/bebraw) directly
 * Send me email at [info@survivejs.com](mailto:info@survivejs.com)
 
-If you post questions to Stack Overflow, tag them using **survivejs** so I will get notified of them. You can use the hashtag **#survivejs** at Twitter.
+If you post questions to [Stack Overflow](http://stackoverflow.com/search?q=survivejs), tag them using **survivejs** so I will get notified of them. You can use the hashtag **#survivejs** at Twitter.
```

Comment:
@wilsonmar @bebraw I think specific link to the tag [`survivejs`](https://stackoverflow.com/questions/tagged/survivejs) is better than the suggested link in the commit.

https://stackoverflow.com/questions/tagged/survivejs